osu.Game/Screens/Play/SubmittingPlayer.cs

```diff
@@ -246,35 +255,13 @@ public override bool Pause()
             return paused;
         }
 
-        protected override void ConcludeFailedScore(Score score)
-        {
-            base.ConcludeFailedScore(score);
-            submitFromFailOrQuit(score);
-        }
-
         public override bool OnExiting(ScreenExitEvent e)
         {
             bool exiting = base.OnExiting(e);
-            submitFromFailOrQuit(Score);
```

Comment:
This removal slightly worries me. I gather that the intent here is that this is moved to `Player.OnExiting()` here:

https://github.com/ppy/osu/blob/a94cdfb1856e4426d637747c8c2aa0f4b7e1116b/osu.Game/Screens/Play/Player.cs#L1176-L1187

Note though that that site has another extra check, namely the `HasPassed` check. Looking at the logic flow that's *maybe* fine still, but I'm not 100% sure. The previous logic somehow gave me more confidence that an exit *will* invariably lead to a submission attempt if the beatmap loaded.

Not requiring any particular action, just one to think about I guess.